docs/building/building-containers.md

```diff
@@ -0,0 +1,100 @@
+# CREATING TRACEE CONTAINER(S) IMAGE(S)
```

Comment:
- We don't use capital letters for page titles and section titles throughout the documentation
- I don't see a landing page to read about different Makefiles and build containers, e.g. docs/building/index.md. Did you forget to update the mkdocs.yml and add it to the main navigation menu?
- We can use shorter paths to have shorter URL -> docs/building/containers.md Otherwise it's http://localhost:8000/tracee/building/building-containers/

Reply:
I did not forget, I didn't do it because I think we had to review everything. So I was including simple markdown pages to be reviewed and indexed later when we do a full docs review.

I'll removing the `building-` prefix from the file names.

Reply:
I have updated mkdocs and will be pushing right after.

Reply:
I have also renamed the files so the path is shorter.